docs/Getting-Started/Create-bootable-media/bootable-usb-using-linux.md

```diff
@@ -0,0 +1,91 @@
+# Create a Bootable Pop!_OS USB in Linux
+
+This section explains the process of creating a bootable Pop!\_OS USB in Linux using Popsicle.
+
+- [Install ISO flashing software.](/Getting-Started/Create-bootable-media/bootable-usb-using-linux.html#install-popsicle)
+- [Download the Pop!_OS.iso file.](/Getting-Started/Create-bootable-media/bootable-usb-using-linux.html#download-the-pop_osiso-file)
+- [Flash the ISO to your USB storage device.](/Getting-Started/Create-bootable-media/bootable-usb-using-linux.html#create-the-bootable-usb)
+
+---
+## Install Popsicle
```

Comment:
There need to be line-breaks surrounding headers, both before  and after. Otherwise our linter will typically flag that with an error.

Reply:
Fixed with db2d943805aef2754a669df39f741dff832311bf.